File: src/main/java/com/dulkirmod/commands/HelpCommand.java
package com.dulkirmod.commands;

import net.minecraft.client.Minecraft;
import net.minecraft.command.CommandException;
import net.minecraft.command.ICommandSender;
import net.minecraft.util.ChatComponentText;
import net.minecraft.util.EnumChatFormatting;

public class HelpCommand extends ClientCommandBase {

    public HelpCommand() {
        super("dulkir");
    }

    @Override
    public void processCommand(ICommandSender sender, String[] args) throws CommandException {
        Minecraft.getMinecraft().thePlayer.addChatMessage(new ChatComponentText(
                EnumChatFormatting.GOLD + "" + EnumChatFormatting.BOLD + " HI THIS IS DULKIRMOD!"));
        Minecraft.getMinecraft().thePlayer.addChatMessage(new ChatComponentText(
                EnumChatFormatting.GRAY + "/enchantrune - toggles enchant rune visibility."));
        Minecraft.getMinecraft().thePlayer.addChatMessage(new ChatComponentText(
                EnumChatFormatting.GRAY + "/fairy - toggles healer fairy visibility."));
    }
}

File: src/main/kotlin/dulkirmod/command/HelpCommand.kt
package dulkirmod.command

import net.minecraft.client.Minecraft
import net.minecraft.command.CommandException
import net.minecraft.command.ICommandSender
import net.minecraft.util.ChatComponentText
import net.minecraft.util.EnumChatFormatting

class HelpCommand : ClientCommandBase("dulkirhelp") {
    @Throws(CommandException::class)
    override fun processCommand(sender: ICommandSender, args: Array<String>) {
        Minecraft.getMinecraft().thePlayer.addChatMessage(
            ChatComponentText(
                EnumChatFormatting.GOLD.toString() + "" + EnumChatFormatting.BOLD + " HI THIS IS DULKIRMOD!"
            )
        )
        Minecraft.getMinecraft().thePlayer.addChatMessage(
            ChatComponentText(
                EnumChatFormatting.GRAY.toString() + "/enchantrune - toggles enchant rune visibility."
            )
        )
        Minecraft.getMinecraft().thePlayer.addChatMessage(
            ChatComponentText(
                EnumChatFormatting.GRAY.toString() + "/fairy - toggles healer fairy visibility."
            )
        )
    }
}